### ISSUE TITLE
Possible incorrect JSON format for u2f requests

### ISSUE THREAD

**Issue by MRuth** [Yubico/u2fval#31]
While testing the v2 branch of the server I am unable to register any u2f devices with a test site running on a local workstation.  I don't know if there is something that needs to be set in the Javascript APIs for registering and authenticating with u2f V2, but there is a major inconsistency between the latest release u2fval server JSON format and the v2 branch format.  Here is a sample of both servers' outputs for a register request.

Latest Release Server v1.0.1
```javascript
{
  "authenticateDescriptors": [],
  "authenticateRequests": [],
  "registerRequests": [
    {
      "challenge": "qaF1VRipNjsxtxmOCs6ggfPkHiQ0iXVPtiIW4wRtg-I",
      "version": "U2F_V2",
      "appId": "https://localhost:9443"
    }
  ]
}
```

V2 branch Server v2.0.0
```javascript
{
  "appId": "https://localhost:9443",
  "descriptors": [],
  "registerRequests": [
    {
      "challenge": "Q5YigLhgKxAw7dNNZ_8jLzCxNIKOvlEEof7e9LXjk9w",
      "version": "U2F_V2"
    }
  ],
  "registeredKeys": []
}
```

**Comment by dainnilsson** [comment]
See the updated documentation in the v2 branch: https://github.com/Yubico/u2fval/tree/v2/doc. Specifically see: https://github.com/Yubico/u2fval/blob/v2/doc/Migrating_From_Older_Versions.adoc

You can also take a look at: https://developers.yubico.com/U2F/Standalone_servers/U2FVAL_REST_API_V2.html although that document is still in the process of being updated.

**Comment by MRuth** [comment]
Okay, I think I have pinpointed the issue.  I am apparently using the U2F v1 client javascript v1 APIs, where this one is calling for the use of the U2F v2 client javascript v1.1 APIs.  However, I have no clue where to obtain those Javascript APIs from.  I have checked the FIDO Alliance page, and only see HTML and PDF links for the Javascript v1.1 guidelines and implementation details, but do not actually see the Javascript v1.1 libraries.

Any advice on where to obtain these?
